submissions/lawalNurudeen/html-forms/contact-form.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Contact-Form</title>
+</head>
+<body>
+    <form>
+        <fieldset>
+            <legend><b>CONTACT US</b></legend>
+            <div>
+                <label for="fullname">Full Name:</label><br>
+                <input type="text" name="fullname" id="fullname" size="60px" placeholder="Enter your full name here">
```

Comment:
Kindly remove the size attribute on all the input field. they are not needed

Reply:
@adaugochi I have just done that..